components/next_steps_view/steps/complete_profile_step.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {StepComponentProps} from '../steps';
+
+type Props = StepComponentProps & {
+};
+
+type State = {
+};
+
+export default class CompleteProfileStep extends React.PureComponent<Props, State> {
+    onSkip = () => {
+        this.props.onSkip(this.props.id);
+    }
+
+    onFinish = () => {
+        this.props.onFinish(this.props.id);
+    }
+
+    render() {
+        return (
+            <div>
+                {'AAAAAAAA'}
+                <div className='NextStepsView__wizardButtons'>
+                    {/* <button
+                        className='NextStepsView__button cancel'
+                        onClick={this.onSkip}
+                    >
+                        <FormattedMessage
+                            id='next_steps_view.skipForNow'
+                            defaultMessage='Skip for now'
+                        />
+                    </button> */}
```

Comment:
All to be replaced in a follow-up PR I assume?

Reply:
Actually @matthewbirtch has asked that I remove it entirely for now until they work out how it affects the UX. I left it commented in case we want to bring it back.

Reply:
What about the `{'AAAAAAAA'}` :P ?

Reply:
Oh that's there just for me :P
That will definitely be replaced.